polyfill/lib/ecmascript.mjs

```diff
@@ -57,7 +58,8 @@ const ES2019 = {
   ToNumber,
   ToObject,
   ToPrimitive,
-  ToString
+  ToString,
+  Type
```

Comment:
This is also in https://github.com/tc39/proposal-temporal/pull/855, if you would like to review that